curriculum/structure/superblocks/back-end-development-and-apis-v9.json

```
@@ -7,7 +7,10 @@
         {
           "dashedName": "introduction-to-nodejs",
           "comingSoon": true,
-          "blocks": ["quiz-node-js-intro"]
+          "blocks": [
+            "review-node-js-intro",
+            "quiz-node-js-intro"
+          ]
```

```suggestion
          "blocks": ["review-node-js-intro", "quiz-node-js-intro"]
```

it's failing here, try this